pkg/server/container_create.go

```diff
@@ -77,6 +86,20 @@ func (c *criContainerdService) CreateContainer(ctx context.Context, r *runtime.C
 	if imageMeta == nil {
 		return nil, fmt.Errorf("image %q not found", image)
 	}
+
+	// Generate container runtime spec.
+	mounts := c.generateContainerMounts(getSandboxRootDir(c.rootDir, sandbox.ID), config)
```

Comment:
Not sure why the cached sandboxID was created.. but it's not being used consistently here.